packages/connect-node-test/src/badweather/broken-input.spec.ts

```diff
@@ -20,130 +20,141 @@ import {
   errorFromJson,
 } from "@bufbuild/connect-core/protocol-connect";
 import { http2Request } from "../helpers/http2-request.js";
-import type { MethodInfo } from "@bufbuild/protobuf";
-import type * as http2 from "http2";
 
 describe("broken input", () => {
   const servers = createTestServers();
   beforeAll(async () => await servers.start());
 
-  // TODO(TCN-785) add @bufbuild/connect-node
-  // TODO(TCN-785) cover gRPC and gRPC-web
-  // TODO(TCN-785) cover invalid protobuf binary input
-  servers.describeServers(["connect-go (h2)"], (server, serverName) => {
-    async function req(method: MethodInfo, type: string, body: Uint8Array) {
-      const url = createMethodUrl(server.getUrl(), TestService, method);
-      const opt: http2.SecureClientSessionOptions = {};
-      if (serverName == "connect-go (h2)") {
-        opt.rejectUnauthorized = false; // TODO set up cert for go server correctly
-      }
-      return await http2Request(
-        "POST",
-        url,
-        { "content-type": type },
-        body,
-        opt
-      );
-    }
+  // TODO(TCN-785) add @bufbuild/connect-node, cover gRPC and gRPC-web, cover invalid protobuf binary input
+  servers.describeServers(
+    ["connect-go (h2)", "@bufbuild/connect-node (h2c)"],
```

Comment:
This enables the tests added in https://github.com/bufbuild/connect-web/pull/350 to run against @bufbuild/connect-node. 

The diff for those files is larger than the actual change, because indentation changed, and the test helper `http2Request()` was updated to take an object argument for readability.

The only actual change here was in the error message expectations, and adding a Content-Type to requests with an unsupported HTTP verb because connect-go and connect-node slightly differ in implementation in this case ( connect-node always returns 415 on Content-Type mismatch, regardless of verb).